compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -135,19 +135,21 @@ fun TooltipArea(
             .onGloballyPositioned { parentBounds = it.boundsInWindow() }
             .onPointerEvent(PointerEventType.Enter) {
                 cursorPosition = it.position
-                startShowing()
+                if (!isVisible && !it.buttons.areAnyPressed) {
+                    startShowing()
+                }
             }
             .onPointerEvent(PointerEventType.Move) {
                 cursorPosition = it.position
-                hideIfNotHovered(parentBounds.topLeft + it.position)
```

Comment:
This doesn't appear needed anymore. Not sure when it was needed.

Reply:
Not sure I understand. The line I removed is in `onPointerEvent(PointerEventType.Move)` - in what scenario is a move received when the pointer is not inside the bounds? The only one I can think of is if the mouse is pressed and dragged outside, but that's handled by the press handler.

> 1 .hover the widget
> 2. wait for the tooltip
> 3. move cursor to the tooltip -> Exit should be triggered on the widget
> 4. we skip hiding of the tooltip in the exit handler (inside hideIfNotHovered)
> 5. move cursor out of the tooltip, but don't hover the widget

In step 5, the cursor is moved from inside the tooltip to outside the tooltip and also outside the tooltip area? If so, it will be hidden as soon as it moves outside the bounds of the tooltip area because of the check in `onPointerEvent(PointerEventType.Move)` on the popup.

Reply:
Okay, I missed the other calls. It seems it should work.